README.md

```diff
@@ -1,9 +1,14 @@
 # Tidy3D
+[![PyPI
+Name](https://img.shields.io/badge/pypi-tidy3d-blue?style=for-the-badge)](https://pypi.python.org/pypi/tidy3d)
+[![PyPI version shields.io](https://img.shields.io/pypi/v/tidy3d.svg?style=for-the-badge)](https://pypi.python.org/pypi/tidy3d/)
+[![Documentation Status](https://readthedocs.com/projects/flexcompute-tidy3ddocumentation/badge/?version=latest&style=for-the-badge)](https://flexcompute-tidy3ddocumentation.readthedocs-hosted.com/?badge=latest)
```

Comment:
Yes in theory. The caveat is that the `latest` docs build branch from that readthedocs repo will be the mirror repository from `tidy3d`. This means that `tidy3d-docs` "main", which points to "latest" will be updated whenever the main `tidy3d` repo gets updated through the sync action. In any case, it's just tweaking how the "latest" branch of that repository gets updated from our sync action, but in any case we have full control to implement that since that badge will just follow the status of the implementation on the corresponding branch.

Reply:
ok sounds good